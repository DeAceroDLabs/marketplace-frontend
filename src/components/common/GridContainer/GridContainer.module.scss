.container {
    display: grid;
    justify-content: left;
    grid-template-columns: auto auto auto;
    max-width: 1100px;
}