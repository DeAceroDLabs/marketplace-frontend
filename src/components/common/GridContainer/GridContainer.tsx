import styles from "./GridContainer.module.scss";

interface GridContainerProps {
  children: React.ReactNode;
}

const GridContainer: React.FC<GridContainerProps> = ({ children }) => {
  return (
    <div>
      <div className={styles.container}>{children}</div>
    </div>
  );
};

export default GridContainer;
