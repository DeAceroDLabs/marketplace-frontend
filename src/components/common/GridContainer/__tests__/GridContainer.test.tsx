import renderer from "react-test-renderer";
import GridContainer from "../GridContainer";

it("renders GridContainer with no issue", () => {
  const component = renderer.create(
    <GridContainer>
      <></>
    </GridContainer>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
