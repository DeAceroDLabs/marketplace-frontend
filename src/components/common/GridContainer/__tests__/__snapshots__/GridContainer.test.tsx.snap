// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders GridContainer with no issue 1`] = `
<div>
  <div
    className="container"
  />
</div>
`;
